### PR TITLE
BZ#1783489 Clarify image registry storage config

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1783489

### CODE REVIEW

**Comment by codyhoag** [comment]
@wzheng1 PTAL?

This PR updates the section for *Image registry storage configuration*. Previews:

AWS UPI and restricted:
https://clarify-image-registry-config--ocpdocs.netlify.com/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-registry-storage-config_installing-aws-user-infra

vSphere/bare metal/IBM-Z:
https://clarify-image-registry-config--ocpdocs.netlify.com/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-registry-storage-config_installing-vsphere

**Comment by codyhoag** [comment]
QE approval in BZ.

File: modules/installation-registry-storage-config.adoc [hunk 1]
```diff
@@ -8,10 +8,35 @@
...
+
 [id="installation-registry-storage-config_{context}"]
 = Image registry storage configuration
 
-If the `image-registry` Operator is not available, you must configure storage
-for it. Instructions for both configuring a PersistentVolume, which is required
+ifdef::aws[]
+Amazon Web Services provides default storage, which means the `image-registry`
+Operator is available after installation. However, if the Registry Operator
+cannot create an S3 bucket, and automatically configure storage, you must
```
Comment (adellape) [inline modules/installation-registry-storage-config.adoc]: Possibly drop comma after bucket?

**Review by adellape — APPROVED** [review]
:+1: LGTM, just one maybe-nit.

File: modules/installation-registry-storage-config.adoc (hunk 1 above)
Comment (codyhoag) [reply modules/installation-registry-storage-config.adoc]: Yeah, a little choppy there. I'll remove it.

**Pull request merged**